realm/realm-library/src/objectServer/java/io/realm/RealmFunctions.java

```diff
@@ -17,30 +17,173 @@
 
 import org.bson.codecs.configuration.CodecRegistry;
 
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import io.realm.internal.Util;
 import io.realm.internal.jni.JniBsonProtocol;
+import io.realm.internal.jni.OsJNIResultCallback;
+import io.realm.internal.objectstore.OsJavaNetworkTransport;
+
+/**
+ * A <i>Realm functions<i> manager to call MongoDB Realm functions.
+ *
```

Comment:
We need to manually add `<p>` tags for empty lines to make the generated Javadoc render correctly. This is also missing in a bunch of other places in this PR.

```suggestion
 * <p>
```